### PR TITLE
DDF-3625 Sanitized Logout Page URL Parameters

### PR DESCRIPTION
**Pull request by idperez**
#### What does this PR do?
Sanitizes a URL parameter on the logout page
#### Who is reviewing it? 
@brjeter 
@blen-desta 
@bakejeyner 
@garrettfreibott 
#### Choose 2 committers to review/merge the PR. 
(please choose ONLY two committers from below, delete the rest)
@andrewkfiedler
@coyotesqrl 
#### How should this be tested? (List steps with links to updated documentation)
Go to: https://localhost:8993/logout/logout-response.html?msg=%3Cscript%3Ealert(1);%3C/script%3E
Verify the view is not corrupted.
#### What are the relevant tickets?
[DDF-3625](https://codice.atlassian.net/browse/)
#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: platform/security/servlet/security-servlet-logout/src/main/webapp/js/logout-response.js [hunk 1]
```diff
@@ -10,7 +10,7 @@
             var param = paramString.split('=');
 
             if (param[0] === "msg") {
-                $('#extramessage').html(decodeURIComponent(param[1].split('+').join(' ')));
+				$('#extramessage').text(decodeURIComponent(param[1].split('+').join(' '))).html();
```
Comment (bdeining) [inline platform/security/servlet/security-servlet-logout/src/main/webapp/js/logout-response.js]: is this formatting correct?

Comment (idperez) [reply platform/security/servlet/security-servlet-logout/src/main/webapp/js/logout-response.js]: hmm good catch!

**Comment by bakejeyner** [comment]
Hero ✔️

**Comment by coyotesqrl** [comment]
build now

**Comment by coyotesqrl** [comment]
build now

**Pull request merged**